mlflow/utils/rest_utils.py

```diff
@@ -153,13 +159,26 @@ class MlflowHostCreds(object):
     :param ignore_tls_verification: If true, we will not verify the server's hostname or TLS
         certificate. This is useful for certain testing situations, but should never be
         true in production.
+        If this is set to true ``server_cert_path`` must not be set.
+    :param client_cert_path: Path to ssl client cert file (.pem).
+        Sets the cert param of the ``requests.request``
+        function (see https://requests.readthedocs.io/en/master/api/).
+    :param server_cert_path: Path to a CA bundle to use.
+        Sets the verify param of the ``requests.request``
+        function (see https://requests.readthedocs.io/en/master/api/).
+        If this is set ``ignore_tls_verification`` must be false.
     """
     def __init__(self, host, username=None, password=None, token=None,
-                 ignore_tls_verification=False):
+                 ignore_tls_verification=False, client_cert_path=None, server_cert_path=None):
         if not host:
             raise MlflowException("host is a required parameter for MlflowHostCreds")
+        if ignore_tls_verification and (server_cert_path is not None):
+            raise MlflowException("if 'MLFLOW_TRACKING_INSECURE_TLS' is set to true "
```

Comment:
@PhilipMay This is a really minor point of feedback, but can we start this error message by indicating that `client_cert_path` and `server_cert_path` should not both be specified and include the environment variable part afterwards as a hint? Users or some other piece of backend code may try to construct this object directly, without the use of environment variables.

Also, can we specify an error code of `INVALID_PARAMETER_VALUE` here? See https://github.com/mlflow/mlflow/blob/master/mlflow/utils/uri.py#L55 for an example.

Reply:
The exception message has been improved. The error code is still missing.

2 lines above my exception is an other `MlflowException` without error code. Since I am not familiar with the proto stuff like in the example I am not sure what to do...
Could you maybe add this after the PR?